components/ui/sonner.tsx

```diff
@@ -0,0 +1,23 @@
+"use client";
+
+import { Toaster as Sonner } from "sonner";
+
+type ToasterProps = React.ComponentProps<typeof Sonner>;
+
+const Toaster = ({ ...props }: ToasterProps) => {
+  return (
+    <Sonner
+      className="toaster group"
+      toastOptions={{
+        duration: 2000,
+        classNames: {
+          toast:
+            "group toast group-[.toaster]:bg-white group-[.toaster]:text-black group-[.toaster]:border-border group-[.toaster]:shadow-lg",
+        },
+      }}
+      {...props}
+    />
+  );
+};
+
+export { Toaster };
```

Comment:
Where exactly are you going to use this component? since i can see in other pages you are directly using sonner library to show error messages.

Reply:
@hassantahir767 , it is used in layout file